### PR TITLE
fix(scss) collapse

### PR DESCRIPTION
**Pull request by almonais**
Remove icon from summary, 
and vertical align the other icon

### CODE REVIEW

File: packages/scss/src/components/_collapse.scss [hunk 1]
```diff
@@ -1,5 +1,9 @@
 .collapse {
 	display: block;
+
+	summary::-webkit-details-marker {
```
Comment (vvalentin-lucca) [inline packages/scss/src/components/_collapse.scss:4]: `.collapse-title` plutôt que `summary` ?

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1229

**Review by vvalentin-lucca — APPROVED** [review]

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1229

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1229

**Pull request merged**